### DISCUSSION TITLE
import fitz

### DISCUSSION THREAD

**Discussion by nishant0001** [pymupdf/PyMuPDF#2550] [Q&A]
hi @JorjMcKie  and all contributor can you tell me is fitz library safe to use for privacy data.
in synk.io it showing vulnerability

**Comment by julian-smith-artifex-com** [comment]
Could you give some more information about the vulnerability?

**Reply by nishant0001** [reply]
sure @julian-smith-artifex-com 
can you go through this link
https://snyk.io/advisor/python/fitz

i have to use pymupdf library on secured private data.
please let me know i can use this library or not?

**Comment by julian-smith-artifex-com** [comment]
The `fitz` package on pypi.org has nothing to do with PyMuPDF.

**Reply by nishant0001** [reply]
thank you so much for reply @julian-smith-artifex-com 
so i can use this library on privacy data?

**Reply by nishant0001** [reply]
@julian-smith-artifex-com  
please tell me using pymupdf on secured privacy pdf is safe or not??

**Reply by jamie-lemon** [reply]
See: https://pymupdf.readthedocs.io/en/latest/tutorial.html#note-on-the-name-fitz

**Reply by JorjMcKie** [reply]
Please reconsider your comment after reading this [blog post](https://artifex.com/blog/pymupdf-1.24.3-and-farewell-to-fitz)!

Since 4 releases PyMuPDF (version 1.24.3) can be imported via its package name. Older import formats can still be used for backward compatibility.
I really don't know what else we can do to protect you from confusion.

That other `fitz` package you mention has never grown out of its alpha version, it is archived and no longer maintained.

PyMuPDF's documentation (installation section) since ever (years before cleaning up in version 1.24.3) is very clear about what can coexist with it in the same Python environment and what not.
Here, too, I cannot imagine what could still be unclear after reading it.

**Comment by nishant0001** [comment]
@julian-smith-artifex-com @JorjMcKie @rk700 @jamie-lemon 
please tell me using pymupdf on secured privacy pdf is safe or not??

**Reply by JorjMcKie** [reply]
Yes, it is! In that sense, PyMuPDF is just a local Document access method. No external dependencies, no internet access, nothing.
